src/main/kotlin/command/SetCurrencyPreference.kt

```
@@ -27,35 +27,38 @@ class SetCurrencyPreference(private val jda: JDA) : Command() {
-            runSQLUntilMaxTries {  userObject = getUserDao().queryForId(event.author.idLong)}
+            runSQLUntilMaxTries { userObject = getUserDao().queryForId(event.author.idLong) }
             if (userObject == null) {
                 userObject = User(event.author.idLong)
             }
             userObject!!.allowCurrencyNotifications = enabled
             runSQLUntilMaxTries {
                 val userDao = getUserDao()
-                userDao.createOrUpdate(userObject)}
+                userDao.createOrUpdate(userObject)
+            }
         } catch (e: DatabaseException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)